README.md

```diff
@@ -1,299 +1,64 @@
-<!-- 
-based on: https://gitlab.gfdl.noaa.gov/fre2/workflows/postprocessing/-/raw/d51df76e537222a3c78405b5749fe59306e6d2bd/README.md
--->
-This repository holds code for defining tasks, applications, tools, workflows, and other aspects of the FRE2 postprocessing
-ecosystem. 
+# fre-workflows
 
+This repository holds code for defining GFDL's next-generation FRE (FMS Runtime Environment) cylc workflows.
+Currently, this repository only contains the `fre-cli` postprocessing workflow.  This documentation will be updated
+when more workflows are added to this repository.
 
-<!-- ______________________________________________________________________________________________________________________ -->
-# Instructions to postprocess FMS history files on GFDL's PP/AN
-These instructions are targeted at workflow developers. If you are a user simply looking to run a specific workflow, see 
-[`fre-cli`](https://github.com/NOAA-GFDL/fre-cli). 
+For more information on cylc workflows please refer to
+[cylc's User Guide](https://cylc.github.io/cylc-doc/stable/html/user-guide/index.html)
 
+## Where to find information
 
+For more information on cylc workflows please refer to
+[cylc's User Guide](https://cylc.github.io/cylc-doc/stable/html/user-guide/index.html)
 
-<!-- ______________________________________________________________________________________________________________________ -->
-## 1. Clone `fre-workflows` repository
-```
-git clone https://github.com/NOAA-GFDL/fre-workflows.git
-cd fre-workflows
-```
-**Do not clone to a temporary directory** - the directory must be readable by slurm from all nodes. Directories on local 
-`\vftmp` are not, while those on `/home`, `/work`, and `/xtmp` are.
+For more information regarding `fre pp` please refer to documentaton in the
+[`fre-cli` repository](https://github.com/NOAA-GFDL/fre-cli)
 
+## What the files are what
 
+| File/directory                    | Purpose |
+| --------------                    | ------- |
+| ```Jinja2Filters```               | Scripts that preprocess the `flow.cylc` file. |
+| ```app/```                        | Tools that are referenced in `flow.cylc` tasks.  These will all be ported to
```

Comment:
Set of workflow task scripts and python modules.  These will be relocated to `fre-cli` in the future.

Reply:
I tried to rework this to be clearer!